rasa/cli/initial_project/domain.yml

```diff
@@ -20,6 +20,10 @@ responses:
   utter_did_that_help:
   - text: "Did that help you?"
 
+  utter_flow_continue_interrupted: 
```

Comment:
@twerkmeister should this be here given that the same is in the default flows file?